tests/unit/test_power_grid_model.py

```diff
@@ -153,7 +154,7 @@ def test_batch_calculation_error_continue(model: PowerGridModel, case_data):
     assert "The id cannot be found:" in error.error_messages[0]
     # assert value result for scenario 0
     result = {"node": result["node"][error.succeeded_scenarios, :]}
-    expected_result = {"node": case_data["output_batch"]["node"][error.succeeded_scenarios, :]}
+    expected_result = {ComponentType.node: case_data["output_batch"]["node"][error.succeeded_scenarios, :]}
```

Comment:
shouldn't `case_data["output_batch"]["node"]` be `case_data["output_batch"][ComponentType.node]` as well? or is that unrelated?

Reply:
@mgovers it could, yes. However, to be truthful, I only updated the tests that dealt directly with internal (non-user accessible) functions of pgm, since now we expect to don't use Literals as component or data types anymore. For the tests that take core of user friendly functions, I mostly left them as is, as they should provide support for both ways and did not think it was really useful to change. If it is desired for me to change all the tests, please let me know.

Reply:
you can leave it, i guess